samples/storage_csm_powerflex_v270.yaml

```diff
@@ -311,3 +311,37 @@ spec:
         # image: Defines replication init container image. This shouldn't be changed
         # Allowed values: string
         image: dellemc/dell-replication-init:v1.0.0          
+
+    - name: resiliency
+      # enabled: Enable/Disable Resiliency feature
+      # Allowed values:
+      #   true: enable Resiliency feature(deploy podmon sidecar)
+      #   false: disable Resiliency feature(do not deploy podmon sidecar)
+      # Default value: false
+      enabled: false
+      configVersion: v1.6.0
+      components:
+        - name: podmon-controller
+          args:
+            - "--csisock=unix:/var/run/csi/csi.sock"
```

Comment:
Can all of these be configured by the user. If not please remove them from the samples

Reply:
Hi @rensyct Some of them are user configurable and some are optional, since these options may be extended in upcoming future so we have kept it here intentionally. At the very beginning even we have kept these under module config but later changes the designed and move it here.

Reply:
Would suggest not to keep anything that are not configurable in the samples. If this is planned for the future, moving the same to samples can be implemented then

Reply:
We have reached a decision to incorporate this as an improvement in the upcoming release. In the meantime, we have included a note in the sample files indicating that certain arguments are not anticipated to be modified by customers.